src/pages/configurePatch/ConfigurePatchCore.tsx

```diff
@@ -93,7 +93,7 @@ export const ConfigurePatchCore: React.VFC<Props> = ({ patch }) => {
     onCompleted(data) {
       const { schedulePatch: scheduledPatch } = data;
       dispatchToast.success("Successfully scheduled the patch");
-      navigate(getVersionRoute(scheduledPatch.versionFull.id));
+      navigate(getVersionRoute(scheduledPatch.versionFull?.id));
```

Comment:
Curious why this was added? I assume that it was necessary to fix a failing test as a result of this upgrade. 
Does this mean https://github.com/evergreen-ci/spruce/pull/1329/files#diff-6deb8e73e58a4a261d3a74d7cf885f9ee14eeefcdd1b156deb268c99e141eb1cR524 maybe didn't work the same way and this was added to fix an error?